pkg/js/validator/validate-dsl.test.ts

```diff
@@ -0,0 +1,45 @@
+import { loadDslSyntaxErrorTestCases } from "../transformer/_testcases";
+import { OpenFgaDslSyntaxMultipleError } from "../transformer/dsltojson";
+import validateDsl from "./validate-dsl";
+
+describe("validateDsl", () => {
+  const testCases = loadDslSyntaxErrorTestCases();
+  testCases.forEach((testCase) => {
+
+    const errorsCount = testCase.expected_errors.length;
+    it(`case ${testCase.name} should throw ${errorsCount} errors on validation`, () => {
+
+      try {
+        validateDsl(testCase.dsl);
+      } catch(thrownError) {
```

Comment:
```suggestion
      } catch(thrownError as OpenFgaDslSyntaxMultipleError) {
```

Can't you do this instead of copying it into an exception below?

If it doesn't work:

```ts
thrownError as any as OpenFgaDslSyntaxMultipleError
```

Reply:
Doesn't like either.

Reply:
We'll deal with it later